Source/animationinfos.cpp

```diff
@@ -122,4 +122,18 @@ void AnimationInfos::SetNewAnimation(uint8_t *Peq, int numFrames, int Delay, Ani
 	}
 }
 
+void AnimationInfos::ProcessAnimation()
+{
+	DelayCounter++;
+	GameTicksSinceSequenceStarted++;
+	if (DelayCounter > DelayLen) {
+		DelayCounter = 0;
+		CurrentFrame++;
+		if (CurrentFrame > FrameLen) {
+			CurrentFrame = 1;
+			GameTicksSinceSequenceStarted = 0;
+		}
+	}
```

Comment:
Would it be possible to use clamp/max/min logic to simplify this block?

Reply:
@julealgon you can mark multiple lines when commenting, that way it's more clear to see what you refer to, especially on the overview page.

Reply:
Ah indeed, forgot about that. Will try to remember it for future comments :S

Reply:
I'm not that familiar with std::clamp but I would say no, cause

- we don't have a upper or lower bound that we can't exceed. If we got the maximum value we start from the beginning (lowest value).
- this helps with debugging as you can set a breakpoint if we get a new frame
- the logic could be advanced/changed if we expand the usage of `AnimationInfo` to other types. So perhaps we could do this in a later cleanup.